src/Hudl.FFmpeg.Core/Attributes/AttributeRetrieval.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
```

Comment:
@rversaw that is correct, I will point out the changes in the PR description shortly